disperser/dataapi/server.go

```diff
@@ -130,6 +135,10 @@ func (s *server) Start() error {
 			feed.GET("/blobs", s.FetchBlobsHandler)
 			feed.GET("/blobs/:blob_key", s.FetchBlobHandler)
 		}
+		operatorsInfo := v1.Group("/operatorsInfo")
+		{
+			operatorsInfo.GET("/deRegisteredLast14days", s.FetchDeRegisteredOperators)
```

Comment:
I would replace this endpoint with something like `/deregistelast/days/14` or `/deregisterlast/14` if it's always days. that way you leave room for extending this ednpoint instead of creating new ones for lets say if we want 30 days

Reply:
or as @QuinnLee suggested days can be a query param.

Reply:
I think Quinn's suggestion is better in this case. Query param. my suggestion (`/deregisterlast/days/14`) doesn't follow rest fully. so update it to use query param